interface/src/lib.rs

```diff
@@ -741,12 +726,14 @@ mod tests {
   #[test]
   fn test_transfer_balance() {
     let mut host = MockHost::new();
+    host.set_balance(&ADDRESS_ALICE, 10000);
     assert_eq!(host.get_balance(&ADDRESS_CHARLIE), 0);
-    assert_eq!(host.get_balance(&ADDRESS_ALICE), SOME_COINS);
+    assert_eq!(host.get_balance(&ADDRESS_ALICE), 10000);
```

Comment:
Why not use the const value?

Reply:
Two reasons:
1. IMHO the `athena-interface` is not the right place for these constants (ADDRESS_ALICE, SOME_COINS, STORAGE_KEY, and so on) and the MockHost implementation. They are not part of the VM interface and shouldn't be exposed by this crate - my "far plan" is to get rid of them all.
2. For me, global constants like these in tests only increase the cognitive load by introducing yet another "name" (that needs to be looked up/jumped to definition to a distant place in code/other file or even crate) to understand its value.

I also removed the state of balances and storage initialized by default in the MockHost to reduce "magic" in tests (state coming out of the blue) - instead the tests start with "empty" state and need to initialize the host with whatever things they need. It leads to tests that easier to understand.